forte2/orbitals/aset.py

```diff
@@ -285,8 +286,19 @@ def _make_embedding(self):
         F_vv = F[np.ix_(virt_inds, virt_inds)]
         lo_vals, Uo = np.linalg.eigh(F_oo)
         lv_vals, Uv = np.linalg.eigh(F_vv)
-        occ_pairs = zip(core_inds, lo_vals)
-        vir_pairs = zip(virt_inds, lv_vals)
+        occ_pairs = sorted(zip(core_inds, lo_vals), key=lambda x: x[1], reverse=True)
+        vir_pairs = sorted(zip(virt_inds, lv_vals), key=lambda x: x[1], reverse=True)
+
+        Ca[:, core_inds] = Ca[:, core_inds] @ Uo
+        Ca[:, virt_inds] = Ca[:, virt_inds] @ Uv
+
```

Comment:
Why do we have two sets of orbital coefficient matrices, Ca and C_orth? If Ca is just old code translated, we should consider changing it to C or something else because I don't think the "a" in Ca is meaningful.